src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -3304,6 +3320,10 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                 tmp = builder->CreateCall(fn_pow, {left_val, right_val});
                 break;
             };
+            default: {
+                throw CodeGenError("Binary operator '" + ASRUtils::binop_to_str(x.m_op) + "' not supported yet",
+                    x.base.base.loc);
+            }
```

Comment:
This is the downside of the current approach: I don't think we should support bit operations on floating point numbers. If you want to do that, you should cast it to integer and then do it. So we should remove the "yet", as we do not plan to support it.

In fact, we need to have two kinds of casts: the current `Cast` transforms the type, such as truncating the float to int. We also need a `Transfer` node, that keeps the bits, just reinterprets the type from float to int.

Reply:
I removed the "yet" part from the error messages. 

@czgdp1807 and I discussed that we should rename `Transfer` (it's confusing!) to something else... `BitCast` might be a good name.